testData/solutions/linux.json

```diff
@@ -377,9 +377,6 @@
         "settingsHandlers": {
             "configuration": {
                 "type": "gpii.orca",
-                "options": {
```

Comment:
Is there no corresponding linux pull request to this? What does the settings handler do if the user token is not supplied?